target_api/settings.py

```diff
@@ -0,0 +1,128 @@
+"""
+Django settings for target_api project.
+
+Generated by 'django-admin startproject' using Django 3.0.7.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.0/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '6=zw0+c5d7h4w^zfj$$z86wu=pcrsfa0z4#8!4kg%^#ef$+=0y'
```

Comment:
Maybe you can add an environment variable here

Reply:
I do it in the next PR, please check #2